src/modules/asab-config/ConfigEditor.js

```diff
@@ -28,6 +28,7 @@ export default function ConfigEditor(props) {
 
 	const [ typeId, setTypeId ] = useState(props.match.params.type_id);
 	const [ type, setType ] = useState(undefined);
+	const [ adHocValue, setAdHocValue ] = useState({});
```

Comment:
`adHocSections`, I guess ... the default should be `[]` ... otherwise I don't get it.